vm/string.go

```diff
@@ -288,6 +288,8 @@ func builtinStringInstanceMethods() []*BuiltInMethodObject {
 			// "Hello\nWorld"[5] # => "\n"
 			// "Hello"[-1]       # => "o"
 			// "Hello"[-6]       # => nil
+			// "Hello😊"[5]      # => "😊"
+			// "Hello😊"[-1]     # => "😊"
```

Comment:
Suggestion: checking '4-byte UTF-8 codepoints' such as '🍣 ' or '🍺' as well is better.
Ref: http://mysqlserverteam.com/sushi-beer-an-introduction-of-utf8-support-in-mysql-8-0/